contracts/src/CrossChainApplications/NativeTokenBridge/INativeTokenSource.sol

```diff
@@ -27,11 +28,12 @@ interface INativeTokenSource {
     event UnlockTokens(address recipient, uint256 amount);
 
     /**
-     * @dev Transfers native tokens to the destination chain as that chain's native token.
+     * @dev Transfers source chain native tokens to destination chain's native tokens.
```

Comment:
Can you make this comment more technically specific? I.e. note what is locked and what message is sent to the destination chain, similar to the description of `transferToSource` in `INativeTokenDestination.sol`